ckanext/ontario_theme/templates/internal/snippets/facet_arrangement.html

```diff
@@ -18,11 +19,10 @@
   {% endfor %}
 {% endif %}
 {% for facet in c.facet_titles %}
```

Comment:
Only facets that are not in `excluded_facets` are actioned (line 28), so in line 21 you could already exclude `facet in c.facet_titles` that are `not in excluded_facets`.

Facets `organization_jurisdiction` and `organization_category` are in `excluded_facets`, so the `choices` variable need not be set i.e. they are never passed to the snippet.

excluded_facets: ['keywords_fr', 'access_level', 'organization', 'organization_jurisdiction', 'organization_category']